src/actions/order-actions.js

```diff
@@ -79,7 +79,8 @@ export function convertCheckoutToOrder (cart, paymentMethod, order) {
   return {
     data: orderPayload,
     payment_method: paymentMethod,
-    card_token: order.cardToken
+    card_token: order.cardToken,
+    paymentAuthorizationID: order.paymentAuthorization.id
```

Comment:
all other keys here seem to be snake_case. could this be changed to payment_authorization_id?